src/components/cylc/Graph.vue

```diff
@@ -633,7 +633,7 @@ export default {
 
   created (cy) {
     console.debug('CREATED')
-    this.workflowId = this.$route.params.workflowid
+    this.workflowId = this.$route.params.workflowName
```

Comment:
The use of ID and Name is somewhat confusing.

Reply:
Agree, and I hadn't really thought too much about it until today.

If users will be able to access each other's workflows, then I think we will have to use IDs everywhere? i.e. kinow/five instead of simply five.

Happy to fix this in this PR too. Just a few lines changes.

Reply:
Users will be able to access other's workflows but only via the other user's UI Server so I think we can just use name.

Reply:
Ah! Of course, we already have a scope with `users/kinow`, no need to do `users/kinow/kinow|five`. Will fix the `Graph` component. Thanks!

Reply:
I take it that is not `/users/<authenticated-user>/<workflow-owner>/five`?

Reply:
(I recall from back in J-Hub testing days, some discussion about two user names in the URL).

Reply:
>I take it that is not /users/<authenticated-user>/<workflow-owner>/five?

That's right, just one user in the URL. I think if we both are running the workflow `five`, then I would be able to access it via

- `http://<server>/users/kinow/#/workflows/five`
- `http://<server>/users/hjoliver/#/workflows/five`

Assuming you granted me permission to access your UI Server/workflow.